app/src/main/res/values/strings.xml

```diff
@@ -140,9 +137,6 @@
     <!-- Title of the screen for adding custom autocomplete URLs. -->
     <string name="preference_autocomplete_title_add">Add custom URL</string>
 
-    <!-- Label for the button to quick add custom autocomplete URLs. -->
-    <string name="custom_autocomplete_quick_add">Add custom URL</string>
```

Comment:
Ah, this string was added for #1933 — It will be used in an upcoming PR

Reply:
So when the PR is merged it will be added back right?